scripts/regression-tests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if ! git diff --name-only $TRAVIS_COMMIT_RANGE | grep -qP '(test/|examples/|package\.json)'
```

Comment:
Should it also include "scripts" and ".travis.yml"?

Reply:
I don't think it should include "scripts", it would make it super hard to test changes to "scripts/regression-tests.sh", because this will always shortcut the script and run all tests.

Maybe the same thing for ".travis.yml"?

Reply:
Maybe just "scripts/reference-tables.*" since that would affect the tests potentially? or is that covered by the "example/" change already?
The travis.yml is probably also an edge case right now, that I hit with #1116 since it potentially affects regression tests, but didn't trigger

Reply:
yeah to test the .travis.yml chances I opened a lot of PRs on the bocoup branch. As for the "reference-tables" -- this produced an html page that isn't tested by the regression tests (because it has not examples on it), so it shouldn't trigger any tests I think!

Reply:
PRs with test files or example files edited, to trigger the tests actually running, is what I mean, [see here](https://github.com/bocoup/aria-practices/pulls)

Reply:
Yeah, I guess you can always temporarily game the build by touching a test file if you want to trigger for .travis.yml changes